README.md

```diff
@@ -2,138 +2,140 @@
-| ----------- | ------------------ | ------------------- | ------------- | ----------------------- | ---------------- | --------------- |
-| Llama-2-7B  | Base (bfloat16)    | 12.212              |  105.14       | 1389.35                 | 13.88            | 13.21           |
-|             | int8dq             | 12.262              |    9.20       |   60.93                 |  8.33            |  6.62           |
-|             | int8wo             | 12.204              |  150.18       |  994.40                 |  8.95            |  6.62           |
-|             | int4wo-64          | 12.843              |  199.86       |  746.66                 |  4.50            |  3.74           |
-|             | int4wo-64-GPTQ     | 12.489              |  199.86       |  746.66                 |  4.50            |  3.74           |
-|             | autoquant          | 12.204              |  159.22       | 1069.87                 |  8.91            |  6.72           |
-| Llama-3-8B  | Base (bfloat16)    | N/A                 |   94.97       | 1425.55                 | 16.43            | 15.01           |
-|             | int8dq             | N/A                 |    8.44       |   63.45                 |  8.98            |  7.52           |
-|             | int8wo             | N/A                 |  139.76       | 1051.02                 | 10.42            |  7.52           |
-|             | int4wo-64          | N/A                 |  179.44       |  757.60                 |  6.62            |  4.22           |
-|             | autoquant          | N/A                 |  137.71       | 1037.74                 | 11.08            |  7.54           |
+We also a developer facing API so you can implement your own quantization algorithms so please use the excellent [HQQ](https://github.com/pytorch/ao/tree/main/torchao/prototype/hqq) algorithm as a motivating example.
 
-note: Int8 dynamic quantization works best on compute bound as opposed to memory bound models. Some relatable examples might be [SAM](https://github.com/pytorch-labs/segment-anything-fast) which is compute bound vs Llama at batchsize=1 which is memory bound.
+### Quantization Aware Training
 
-For int4 we make heavy use of [tinygemm](https://github.com/pytorch/ao/blob/cb3bd8c674f2123af232a0231b5e38ddafa756a8/torchao/dtypes/aqt.py#L526) of `torch.ops.aten._weight_int4pack_mm` to bitpack into a layout optimized for tensor cores
+Quantizing a model in this way is called Post Training Quantization and it can sometimes give you a fast, small but innacurate model. To fix this we recommend you try out Quantization Aware Training and we shared a recipe in collaboration with the torchtune team [here](https://pytorch.org/blog/quantization-aware-training/) where we recovered up to **96% of the accuracy degradation on hellaswag and 68% of the perplexity degradation on wikitext** for Llama3 compared to post-training quantization (PTQ). 
```

Comment:
suggest rephrasing this to something like 
"Using post-training quantization can result in a fast and compact model, but may also lead to accuracy degradation. To overcome this limitation, we recommend exploring Quantization Aware Training (QAT). In collaboration with Torchtune, we've developed a QAT recipe that demonstrates significant accuracy improvements over traditional PTQ, recovering..."